vmas/simulator/dynamics/drone.py

```diff
@@ -0,0 +1,128 @@
+from typing import Union
+
+import torch
+import vmas.simulator.core
+import vmas.simulator.utils
+from vmas.simulator.dynamics.common import Dynamics
+
+
+class Drone(Dynamics):
+    def __init__(
+        self,
+        world: vmas.simulator.core.World,
+        mass: float = 0.2,
+        I_xx: float = 8.1e-3,
+        I_yy: float = 8.1e-3,
+        I_zz: float = 14.2e-3,
+        integration: str = "rk4",
+    ):
+        super().__init__()
+
+        assert integration in (
+            "rk4",
+            "euler",
+        )
+
+        self.mass = mass
+        self.I_xx = I_xx
+        self.I_yy = I_yy
+        self.I_zz = I_zz
+        self.world = world
+        self.g = 9.81
+        # Drone state: phi(roll), theta (pitch), psi (yaw),
+        #              p (roll_rate), q (pitch_rate), r (yaw_rate),
+        #              x_dot (vel_x), y_dot (vel_y), z_dot (vel_z),
+        #              x (pos_x), y (pos_y), z (pos_z)
+        self.drone_state = torch.tensor(
+            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0]
+        )
```

Comment:
So this should have the device of the world and the first batched dimension.

You can do it with 
```python
self.drone_state = torch.tensor(
            [0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 0.0, 1.0], device=world.device
        ).unsqueeze(0).expand(world.batch_dim, 12)
```